packages/commonwealth/client/scripts/views/components/chat/chat_window.tsx

```diff
@@ -37,17 +38,50 @@ type ChatWindowAttrs = {
 };
 
 export class ChatWindow implements m.Component<ChatWindowAttrs> {
-  private onIncomingMessage: (any: any) => void;
-  private scrollToBottom: () => void;
-  private shouldScroll: boolean;
-  private shouldScrollToHighlight: boolean;
-  private quillEditorState: any;
+  onIncomingMessage: (any: any) => void;
```

Comment:
We should make these private unless they're externally exposed.

Reply:
Yes! Will update. Think merge overwrote.